### PR TITLE
docs: build sllm-store from source

### PR DESCRIPTION
**Pull request by SiyangShao**
# Overview
This pull request adds README for how to build `sllm-store` from source.

### CODE REVIEW

File: serverless_llm/store/README.md [hunk 1]
```diff
@@ -1 +1,37 @@
-# serverless_llm-store
\ No newline at end of file
+# serverless_llm-store
+
+## About
+
+`sllm-store` is an internal library of ServerlessLLM that provides high-performance model loading from local storage into GPU memory. You can also install and use this library in your own projects, following our [quick start guide](https://serverlessllm.github.io/docs/stable/store/quickstart).
+
+## Build From Source
```
Comment (future-xy) [inline serverless_llm/store/README.md:7]: This is the way to build the wheel. Might be over-complex for local development. I suggest just using `pip install .`

File: serverless_llm/store/README.md [hunk 2]
```diff
@@ -1 +1,37 @@
-# serverless_llm-store
\ No newline at end of file
+# serverless_llm-store
+
+## About
+
+`sllm-store` is an internal library of ServerlessLLM that provides high-performance model loading from local storage into GPU memory. You can also install and use this library in your own projects, following our [quick start guide](https://serverlessllm.github.io/docs/stable/store/quickstart).
```
Comment (future-xy) [inline serverless_llm/store/README.md:5]: Could you also please update the docs?

Comment (future-xy) [reply serverless_llm/store/README.md:5]: I mean add an "install from source" subsection.

**Review by future-xy — CHANGES_REQUESTED** [review]

File: serverless_llm/store/README.md (hunk 2 above)
Comment (SiyangShao) [reply serverless_llm/store/README.md:5]: Updated that under "install with pip"

File: serverless_llm/store/README.md (hunk 1 above)
Comment (SiyangShao) [reply serverless_llm/store/README.md]: Added way to just using `pip install .`

**Review by future-xy — APPROVED** [review]

**Pull request merged**